### ISSUE TITLE
Support pass/fail verdicts in algorithms that compare reports to determine which tests have equivalent results

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#736]
This issue is one element of the plan documented in w3c/aria-at#961 to migrate from 3 assertion verdicts (good/no output/incorrect output) to two verdicts (pass/fail).

**Comment by ccanash** [comment]
Hi @mcking65  this was included in dec 5th release. Can it be closed?
